### ISSUE TITLE
aiohttp has moved MultiDict maybe import from multidict package instead? 

### ISSUE THREAD

**Issue by graingert** [aio-libs/aiobotocore#137]
```pytb
Traceback (most recent call last):
  File "./redacted.py", line 4, in <module>
    import aiobotocore
  File "/opt/venv/lib/python3.5/site-packages/aiobotocore/__init__.py", line 1, in <module>
    from .session import get_session, AioSession
  File "/opt/venv/lib/python3.5/site-packages/aiobotocore/session.py", line 7, in <module>
    from .client import AioClientCreator
  File "/opt/venv/lib/python3.5/site-packages/aiobotocore/client.py", line 11, in <module>
    from .args import AioClientArgsCreator
  File "/opt/venv/lib/python3.5/site-packages/aiobotocore/args.py", line 9, in <module>
    from .endpoint import AioEndpointCreator
  File "/opt/venv/lib/python3.5/site-packages/aiobotocore/endpoint.py", line 14, in <module>
    from aiohttp import MultiDict
ImportError: cannot import name 'MultiDict'
```

**Comment by jettify** [comment]
yes, aiohttp updated to 2.0.0 (released today) with backward incompatible changes. Fix require more then just moving MultiDict, if you are interested see my testing branch here https://github.com/aio-libs/aiobotocore/pull/135/files

**Comment by graingert** [comment]
Fixed by #139

https://github.com/aio-libs/aiobotocore/commit/e06c29d8e36f04a7981fce0a13387e54ec05c524

**Comment by jettify** [comment]
created issue to track aiohttp support https://github.com/aio-libs/aiobotocore/issues/140
